.github/ISSUE_TEMPLATE.md

```diff
@@ -29,6 +29,10 @@ https://github.com/nvaccess/nvda/wiki/Github-issue-template-explanation-and-exam
 #### Other information about your system:
 
 ### Other questions
-#### Does the issue still occur after restarting your PC?
+#### Does the issue still occur after restarting your machine?
 
 #### Have you tried any other versions of NVDA? If so, please report their behaviors.
+
+#### If addons are disabled, is your problem still occuring?
+
+#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
```

Comment:
I'm not sure about adding this one. While there may not be any negative consequence to running this, adding this advice means that it's harder to know how often this actually fixes an issue. I get the feeling this is suggested rarely in the first place.